io.github.niccokunzmann.python_dhcp_server.yml

```diff
@@ -0,0 +1,87 @@
+id: io.github.niccokunzmann.python_dhcp_server
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: python_dhcp_server
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --allow=multiarch
+  - --persist=/app/config
```

Comment:
you don't understand the sandbox and what persist does.

If it's because you expect to be able to write in /app, then your expectation is incorrect.

Reply:
Oh, yes, I was wondering why I could not write to that directory. I saw a bit of documentation but I really just would like to store the state of the app.

Reply:
Ahhh. Now, I understand... I looked at the documentation for a while and suggested an edit for newbies. It persists now :tada:

Reply:
the suggestion is incorrect.

it is incorrect to use `/var`